packages/angular_devkit/build_angular/src/dev-server/index.ts

```diff
@@ -243,7 +242,14 @@ export class DevServerBuilder implements Builder<DevServerBuilderSchema> {
     } catch {
       throw new Error('The "webpack-dev-server" package could not be found.');
     }
-    const entryPoints = [`${webpackDevServerPath}?${clientAddress}`];
+
+    // If a custom path is provided the webpack dev server client drops the sockjs-node segment.
+    // This adds it back so that behavior is consistent when using a custom URL path
+    if (clientAddress.pathname) {
+      clientAddress.pathname = path.posix.join(clientAddress.pathname, 'sockjs-node');
+    }
+
+    const entryPoints = [`${webpackDevServerPath}?${url.format(clientAddress)}`];
```

Comment:
Any reason why we don't use `addDevServerEntrypoints`?
https://github.com/webpack/webpack-dev-server/blob/21687c3bfc95752957d16470b97d281001dc6a07/lib/Server.js#L971
https://github.com/webpack/webpack-dev-server/blob/21687c3bfc95752957d16470b97d281001dc6a07/lib/utils/addEntries.js#L6

Reply:
Historical mostly.

I was intending this to be minimal so that it could go on patch as well.
Once the update builders are in place, I'd like to go through each one and cleanup/refactor where possible.